### DISCUSSION TITLE
two-pane view

### DISCUSSION THREAD

**Discussion by santa2021** [antonmedv/llama#74] [Ideas]
Hello, is it possible to implement a two-pane view in your project, as it is done for example here:https://github.com/anmitsu/goful

**Comment by antonmedv** [comment]
I’d like to keep llama simple. I didn’t like other file managers as they offer way too many features. 

And creating panes is easy with tmux or iterm2. Just create a panel and start llama there.

**Reply by santa2021** [reply]
Tmux is a multiplexer, not a file manager, so your recommendation is a bit out of place. Well, maybe someone else implements the things I need in the file manager. Good luck

**Reply by antonmedv** [reply]
I will definitely not merge it 😏

**Reply by ghost** [reply]
> Tmux is a multiplexer, not a file manager, so your recommendation is a bit out of place. Well, maybe someone else implements the things I need in the file manager. Good luck

the entitlement 💀

**Reply by tayopi** [reply]
I understand the point @antonmedv is making and walk is not the only file manager that follows this  philosophy. LF also does not support a two pane view and they offer the same solution to users that want a two pane view. Just use a multiplexer and open the file manager application in both.
